.travis.yml

```diff
@@ -94,7 +94,7 @@ deploy:
 
 before_install:
   # Run pylint first thing, and fail fast!
-  - ". ./ci/travis/run_pylint.sh"
+  # - ". ./ci/travis/run_pylint.sh"
```

Comment:
If you're going to leave this in the comment commented out, then I recommend adding a comment in the code explaining why, and including a github issue # that tracks it.